node/text-to-speech-with-huggingface/src/main.js

```diff
@@ -0,0 +1,47 @@
+import { HfInference } from '@huggingface/inference';
+import { getStaticFile, throwIfMissing } from './utils.js';
+
+export default async ({ req, res }) => {
+  throwIfMissing(process.env, ['HUGGINGFACE_ACCESS_TOKEN']);
```

Comment:
Is this ACCESS_TOKEN or API_TOKEN? In the readme for this function I see API_TOKEN. 
Please search through the other examples as well to make sure we use the same variable

Reply:
I've standardised this to ACCESS_TOKEN - this is how Hugging Face refer it on their docs